packages/node/test/handlers.test.ts

```diff
@@ -164,4 +171,133 @@ describe('parseRequest', () => {
       expect(parsedRequest.transaction).toEqual('routeHandler');
     });
   });
-});
+}); // end describe('parseRequest()')
+
+describe('tracingHandler', () => {
+  const urlString = 'http://dogs.are.great:1231/yay/';
+  const queryString = '?furry=yes&funny=very';
+  const fragment = '#adoptnotbuy';
```

Comment:
<3

Reply:
> When do we even have the case when we have too many slashes? Afaik ClientRequest is printing http or https, without any slashes

If it's an internal route, there's no protocol or hostname, and it looks like this:

![image](https://user-images.githubusercontent.com/14812505/91860569-adf75e80-ec20-11ea-9d86-b0fd4bf07ec2.png)